using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,403 @@
+3. Within the third request we pass the 2nd token to the vendor API once again. For this request, we will receive a last token that can be used to fetch actual data from the API.
+
+!(Pasted image 20240214224605.png)
+
+As all of these tokens will expire after some time, it could be possible to save the tokens and reuse them instead of chaining all the requests for every run of the flow. For a simple way to setup we run it within a single Flow and pass each token to the next operation. 
+
+To pass data from one Flow Operation to another Operation, the `{{ … }}` syntax can be used. With the `{{ … }}` you can refer to any object that is currently part of the Flow. Here the functionality of named Operation keys becomes handy to easily identify and maintain the Flow.
+
+> [!TIP] Name your Flow operations
+> ToDo why this could be a best practice.
+> - Easy to apply
+> - easy to use and to remember of $last is not an option
+
+The screenshot shows the last request. As a key, we've set `token_sign_in` which can be used in the next operation. Within the payload/request body, we've specified the token to use with the dynamic code `"{{$last.data.mini_token}}"`. The object `$last` refers to the previous operation, an easy way to access the results from the last request. 
+
+!(Pasted image 20240216105507.png)
+ 
+##### Getting actual user data
+Compared to the auth process, fetching actual data from the API is done with fewer requests – just two are needed. As the vendor supports multiple devices, we have to get a list of devices first. In theory, we can also do this in preparation and hard-code the device ID within the next operation. But again, as there is no official documentation of the API this might change or the ID (which is more of a name) might not that unique and may change within any future updates. 
+
+So let's catch the device name, or `dsn` as it is called by the external API. Again we are using a Web request with the last token we got from the auth-process.
+
+The GET URL is static and does not contain any dynamic value this time.
+
+| Method | URL |
+| --- | --- |
+| GET | https://ads-field-eu-1a2039d9.aylanetworks.com/apiv1/devices.json |
+As header parameter, we set the following:
+
+| Header | Value |
+| --- | --- |
+| Authorization | `auth_token {{$last.data.access_token}}` |
+As you can see, we use the `{{ … }}` to pass the access_token from the last auth request.
+
+If everything works so far, the JSON we receive looks something like this, but for security reasons, some data was removed or masked as it also contains coordinates, MAC addresses, IP addresses and so on:
+```json
+{
+  "status": 200,
+  "statusText": "OK",
+  "headers": {
+    ...
+  },
+  "data": [
+    {
+      "device": {
+        "dsn": "AC00***697",
+        "lan_ip": "192.168.*.*",
+        "lan_enabled": false,
+        "has_properties": true,
+        "connection_status": "Online",
+        "lat": "xxxxxxxx",
+        "lng": "xxxxxxxx",
```

Comment:
Can you replace these with *** so it's clear that this character means redaction?